stats/lib/platform-stats-fetchers.js

```diff
@@ -43,6 +43,20 @@ export const fetchDailyStationAcceptedMeasurementCount = async (pgPool, filter)
   return rows
 }
 
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchTopMeasurementParticipants = async (pgPool, filter) => {
+  // Ignore the filter for this query
```

Comment:
> When I request top participants using a date range for today, the API will return the response using data from yesterday. 

I think this can't happen because if filter=today then the assert above will throw

Reply:
ah I see this comment is old. Will resolve